integrations/aws/main.py

```diff
@@ -0,0 +1,184 @@
+    serializable_account_accounts = []
+    for account in describe_accessible_accounts():
+        serializable_account_accounts.append(_fix_unserializable_date_properties(account))
+    yield serializable_account_accounts
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.CLOUDRESOURCE)
+async def resync_generic_cloud_resource(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    resource_kinds = get_resource_kinds_from_config(kind)
+
+    if len(resource_kinds) == 0:
+        logger.error(f"Resource kinds not found in port app config for {kind}")
+        return
+
+    for kind in resource_kinds:
+        async for batch in resync_cloudcontrol(kind):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.ACM)
+async def resync_acm(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in _get_sessions():
+        session = await session
+        async for batch in batch_resources(kind, session, 'acm', 'list_certificates', 'CertificateSummaryList', 'NextToken'):
+            yield batch
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.LOADBALANCER)
+async def resync_loadbalancer(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    sessions = _get_sessions()
+    async for session in sessions:
+        async for batch in batch_resources(kind, session, 'elbv2', 'describe_load_balancers', 'LoadBalancers', 'NextMarker'):
+            yield batch
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.CLOUDFORMATION)
+async def resync_cloudformation(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in _get_sessions():
+        async for batch in batch_resources(kind, session, 'cloudformation', 'list_stacks', 'StackSummaries', 'NextToken'):
+            yield batch
+
+
+async def resync_cloudcontrol(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    is_global = is_global_resource(kind)
+    async for session in _get_sessions(None, DEFAULT_REGION if is_global else None):
+        region = session.region_name
+        logger.info(f"Resyncing {kind} in region {region}")
+        account_id = await find_account_id_by_session(session)
+        next_token = None
+        while True:
+            all_instances = []
+            try:
+                async with session.client("cloudcontrol") as cloudcontrol:
+                    params = {
+                        'TypeName': kind,
+                    }
+                    if next_token:
+                        params['NextToken'] = next_token
+                    
+                    response = await cloudcontrol.list_resources(**params)
+                    next_token = response.get('NextToken')
+                    resources = response.get('ResourceDescriptions', [])
+                    if not resources:
+                        break
+                    for instance in response.get('ResourceDescriptions', []):
+                        described = await describe_single_resource(kind, instance.get('Identifier'), account_id, region)
```

Comment:
if you are describing each wont it be tons of requests?

Reply:
still waiting for a response

Reply:
Yes it will but it's either this or losing properties